README.md

```diff
@@ -10,6 +10,8 @@ This directory contains everything that allows us to create a Docker image with
 
 Currently, our base image is Ubuntu, as we require glibc 2.33+.
 
+It's currently pushing the resuting images to: https://hub.docker.com/r/timescaledev/timescaledb-ha
```

Comment:
```suggestion
It's currently pushing the resulting images to: https://hub.docker.com/r/timescale/timescaledb-ha
```

Reply:
I think we push to `/timescale/` since commit 0c54000b3eb213fffc96d37ba98d9758a6adbddf

Reply:
yup, is pushing to timescaledev just so that it can run concurrently with the master branch, and would be a thing that changes just before merging.